trace/opencensus/trace/tracer/base.py

```diff
@@ -81,3 +84,17 @@ def add_label_to_spans(self, label_key, label_value):
     def list_collected_spans(self):
         """List collected spans."""
         raise NotImplementedError
+
+
+class NullObject(object):
```

Comment:
Can we call this `NullContextManager`?  `NullObject` is a little non-specific.

Reply:
Done.